docs/launch-with/anvil-launch-with.rst

```diff
@@ -45,12 +45,7 @@ Limitations
 
 1. While we support launching of WDL workflows, tools as listed in
    Dockstore are currently not supported.
-2. AnVIL does not currently support file-path based imports. Importing a
-   workflow with file-based imports will result in error. See the
-   :ref:`converting file-based imports
-   doc <converting-file-path-based-imports-to-public-http-s-based-imports-for-wdl>`
-   for more info.
-3. Only the WDL language is supported.
+2. Only the WDL language is supported.
 
```

Comment:
Do you want to add that parameter files (json) are not included in the launch with? 

3. Example parameter files (json) are not included in the launch-with service. After a workflow launch is complete, users can download parameter files from Dockstore and upload them into their Terra workspace that contains the workflow.

Reply:
> Do you want to add that parameter files (json) are not included in the launch with?
> 
>     1. Example parameter files (json) are not included in the launch-with service. After a workflow launch is complete, users can download parameter files from Dockstore and upload them into their Terra workspace that contains the workflow.

Hmm, I'll see what others think... If we add it, I don't think it should be a limitation, but I could see it being useful in the general overview of how to import.

Reply:
Could be good to add as a tip or something like that on the page

Reply:
Added tip